charts/posthog/values.yaml

```diff
@@ -417,10 +417,10 @@ kafka:
     # -- Enable persistence using PVC
     enabled: true
     # -- PVC Storage Request for kafka volume
-    size: 5Gi
+    size: 20Gi
```

Comment:
as a post merge follow up, we can probably remove all the `kafka.size` overrides we added in the docs as initial mitigation

Reply:
yup, that's the plan & part of https://github.com/PostHog/posthog.com/pull/2411